great_expectations/init_notebooks/create_expectations.ipynb

```diff
@@ -106,7 +142,7 @@
     "\n",
     "```\n",
     "data_asset_name = 'CHOOSE THE NAME OF YOUR TABLE FROM THE LIST OF DATA ASSETS ABOVE'\n",
-    "df = context.get_batch(data_asset_name, \n",
+    "batch = context.get_batch(data_asset_name, \n",
     "                        expectation_suite_name='my_suite'\n",
     "                        BatchKwargs(table=data_asset_name)) \n",
```

Comment:
After seeing this used earlier, I think it might be dangerous to suggest to people to use generic BatchKwargs this early in the process.

What about changing BatchKwargs(table=data_asset_name) to:
1) ensure we're in the world of the table generator, then
2) `context.build_batch_kwargs(data_asset_name, partition_id=table_name)`

Reply:
I want to make into a separate task, since this text requires another round of reviews.